bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -160,7 +163,7 @@ impl Timeline {
         self.inner.fetch_members().await
     }
 
-    pub fn subscribe_to_back_pagination_status(
+    pub async fn subscribe_to_back_pagination_status(
```

Comment:
It doesn't seem necessary to have this method `async`. Am I missing something?

Reply:
Indeed, that's a remnant of a previous impl. Thanks!